pkg/graph/schema.graphql

```diff
@@ -1943,6 +1999,7 @@ type Mutation {
   createTRBRequestDocument(input: CreateTRBRequestDocumentInput!): CreateTRBRequestDocumentPayload
   deleteTRBRequestDocument(id: UUID!): DeleteTRBRequestDocumentPayload
   updateTRBRequestForm(input: UpdateTRBRequestFormInput!): TRBRequestForm!
+  createTRBRequestFeedback(input: CreateTRBRequestFeedbackInput!): TRBRequestFeedback!
```

Comment:
We usually have a separate Payload type that wraps the returned data (see `CreateSystemIntakeContactPayload`, for instance), though we're not 100% consistent on that. Should we do that here, have a `CreateTRBRequestFeedbackPayload` type that wraps `TRBRequestFeedback`?

Reply:
I'm following the pattern that @StevenWadeOddball introduced when we started using patterns that are more like the MINT ones

Reply:
Ok, that makes sense. I'll keep that in mind for the future.